scipy/interpolate/_rbfinterp.py

```diff
@@ -10,27 +10,17 @@
 
 from ._rbfinterp_pythran import _build_system, _evaluate, _polynomial_matrix
 
-
 __all__ = ["RBFInterpolator"]
 
-
 # These RBFs are implemented.
 _AVAILABLE = {
-    "linear",
-    "thin_plate_spline",
-    "cubic",
-    "quintic",
-    "multiquadric",
-    "inverse_multiquadric",
-    "inverse_quadratic",
-    "gaussian"
-    }
-
+    "linear", "thin_plate_spline", "cubic", "quintic", "multiquadric",
+    "inverse_multiquadric", "inverse_quadratic", "gaussian"
+}
 
 # The shape parameter does not need to be specified when using these RBFs.
 _SCALE_INVARIANT = {"linear", "thin_plate_spline", "cubic", "quintic"}
```

Comment:
You probably used autoformating on the whole file. Please don't as it not good for the history and does not facilitate reviews. If there are clear pep8 violations around the changes it's fine, but not more like this one for instance where it's very subjective.

Reply:
Yes, that was my mistake, I've forgot to turn yapf off, and then it too late with too many changes in the file.

Reply:
Please revert these.

Reply:
Thank you 🙏